Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFSectionTitlesViewController.h

```diff
@@ -0,0 +1,16 @@
+//  Created by Monte Hurd on 7/31/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <Foundation/Foundation.h>
+
+@interface WMFSectionTitlesViewController : NSObject
+
+@property (nonatomic, strong) UIWebView* webView;
```

Comment:
Can we init with the needed properties instead of setting them later? Makes it easier to test when we inject on initialization and make them immutable.